src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
+  --color: #00acdc;
+
+  font-family: Roboto, serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link {
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
```

Comment:
Using a specific color like 'black' can be inflexible. Consider using a CSS variable or a more flexible color scheme to allow for easier theme changes in the future.